Common/Sources/View/Error/ErrorView.swift

```diff
@@ -17,21 +17,33 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 import HyperwalletSDK
+import Insights
 import UIKit
 
 /// The class to handle UI errors
 public final class ErrorView {
-    weak var viewController: UIViewController!
-    var error: HyperwalletErrorType
+    private let errorTypeConnection = "CONNECTION"
+
+    private weak var viewController: UIViewController!
+    private var error: HyperwalletErrorType
+    private var pageName: String?
+    private var pageGroup: String?
```

Comment:
These should not be optional and should be passed from all the Controllers

Reply:
fixed